prow/config.yaml

```diff
@@ -81,13 +81,17 @@ branch-protection:
           required_status_checks:
             contexts:
             - check-pr-has-kind-label
-            - "e2e tests (k8s-oldest, alpha)"
-            - "e2e tests (k8s-oldest, beta)"
-            - "e2e tests (k8s-oldest, stable)"
-            - "e2e tests (k8s-plus-one, alpha)"
-            - "e2e tests (k8s-plus-one, beta)"
-            - "e2e tests (k8s-plus-one, stable)"
-            - "lint"
+            - "ci / build"
+            - "ci / test"
+            - "ci / lint"
+            - "ci / Check generated code"
+            - "ci / Multi-arch build"
+            - "ci / e2e-tests / e2e tests (k8s-oldest, alpha)"
+            - "ci / e2e-tests / e2e tests (k8s-oldest, beta)"
+            - "ci / e2e-tests / e2e tests (k8s-oldest, stable)"
+            - "ci / e2e-tests / e2e tests (k8s-plus-one, alpha)"
+            - "ci / e2e-tests / e2e tests (k8s-plus-one, beta)"
+            - "ci / e2e-tests / e2e tests (k8s-plus-one, stable)"
```

Comment:
Should these just use the job name without the  `ci /` or `ci / e2e-tests /` prefix, the same as they would be configured in the GitHub branch protection rules?

e.g. `e2e tests (k8s-plus-one, stable)`

Reply:
Ah you are right, the `ci` is probably not required.

![2025-02-19-061742](https://github.com/user-attachments/assets/299ead9c-964b-445c-9ba0-813451326560)